packages/queries/src/schema/getPrimaryKey.ts

```diff
@@ -0,0 +1,33 @@
+import { DbSchema } from '../types/DbSchema';
+import { Table } from '../types/Table';
+import { getColumnDefs, getTableDef, isPrimaryKeyColumn } from './getTableDefs';
+
+/**
+ * Get the primary key of a table, as defined in the database schema.
+ *
+ * Example:
+ *
+ * ```ts
+ * const primaryKey = getPrimaryKey<DB>(schema, 'actor');
+ *
+ * console.log(primaryKey); // Prints 'actor_id'
+ * ```
+ *
+ * @param schema The generated schema object for the database
+ * @param table The name of the table
+ */
+
+export function getPrimaryKey<DB>(
+    schema: DbSchema<DB>,
+    table: Table<DB>,
+): string {
```

Comment:
this isn't correct: this should return a `string[]` as multiple columns can be part of the primary key

Reply:
True that.. fixing now